### ISSUE TITLE
Installation via NuGet

### ISSUE THREAD

**Issue by abjerner** [mindrevolution/SirTrevor-for-Umbraco#19]
Personally I prefer installing as much as possible via NuGet. The Umbraco package will create a new data type, but the rest is just files, which also can be achieved quite easily through a NuGet package.

**Comment by MarcStoecker** [comment]
True. I was planning on providing that with the new build script that I have started. Idea is to generate umb package and nuget package together, in sync.
